.github/workflows/build-fedora.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -eux
+set -o pipefail
+
+# shellcheck disable=SC2206
+PHASES=(${@:-DEPS INITRD})
+
+for phase in "${PHASES[@]}"; do
+    case "$phase" in
+        DEPS)
+            echo "Installing necessary dependencies"
+            dnf -y install dnf-plugins-core kernel-core mkosi python3-pyxattr systemd zstd
+            ;;
+        INITRD)
+            rm -fr mkosi.output
+
+            # Can't use `uname -r`, since we're in a container
+            KVER="$(rpm -q kernel-core --qf "%{version}-%{release}.%{arch}\n" | sed 1q)"
+            [[ -e "kernel-core-$KVER.rpm" ]] || dnf -y download "kernel-core-$KVER"
```

Comment:
This shouldn't be necessary, `mkosi.finalize` downloads the image automatically now.

Reply:
Ah, interesting, didn't know about that. I'll try it with the next batch of CI updates.